### PR TITLE
Fix/search 1444 crlf

### PR DESCRIPTION
**Pull request by alessandrobenedetti**
Content Store testing improved and line endings fixed

### CODE REVIEW

**Comment by agazzarini** [comment]
Internal review done. LGTM

File: alfresco-search/src/test/java/org/alfresco/solr/content/SolrContentStoreTest.java [hunk 1]
```diff
@@ -36,31 +36,87 @@
 import org.mockito.Mockito;
 import org.mockito.runners.MockitoJUnitRunner;
```
Comment (tunaaksoy) [inline alfresco-search/src/test/java/org/alfresco/solr/content/SolrContentStoreTest.java]: We could have updated the deprecated class with org.mockito.junit.MockitoJUnitRunner :)

**Comment by tunaaksoy** [comment]
Sorry, I might be missing something but what is the fix here?

File: alfresco-search/src/test/java/org/alfresco/solr/content/SolrContentStoreTest.java (hunk 1 above)
Comment (alessandrobenedetti) [reply alfresco-search/src/test/java/org/alfresco/solr/content/SolrContentStoreTest.java]: Done!

**Comment by alessandrobenedetti** [comment]
Old tests were incomplete and were creating a temp folder null/contentstore that was never cleaned from the filesystem.
Was just a test improvement, no actual code change except the logging

**Comment by tunaaksoy** [comment]
LTGM and can be merged.

**Pull request merged**